README.md

```diff
@@ -94,8 +94,10 @@ LaTeX.js can be used as a web component:
 
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-  <script src="https://cdn.jsdelivr.net/npm/latex.js@0.11.1/dist/latex.min.js"></script>
-  <script src="https://cdn.jsdelivr.net/npm/latex.js@0.11.1/dist/latex.component.js"></script>
+  <script type="module">
+  import latexjs from "https://cdn.jsdelivr.net/npm/latex.js/dist/latex.component.js"
+  customElements.define('latex-js', latexjs)
+  </script>
```

Comment:
running `npm test` fails for the component, so you also need to update `test/api/webcomponent.html`...